### PR TITLE
Using wget, as there are libcurl3 dependencies and curl needs libcurl4

### PR DESCRIPTION
**Pull request by AnandShastry**
Using wget, as there are libcurl3 dependencies and curl needs libcurl4

### CODE REVIEW

**Comment by AnandShastry** [comment]
Please check this pull request. This change is specifically made to address libcurl3 dependency. One of the shared object in dot net tools need libcurl3 and curl needs curl4 library.

**Review by AnandShastry — COMMENTED** [review]
@radical Can you please approve pull request?

File: cibuild.sh [hunk 1]
```diff
@@ -39,8 +39,9 @@ downloadMSBuildForMono()
         mkdir -p "$PACKAGES_DIR" # Create packages dir if it doesn't exist.
 
         echo "** Downloading MSBUILD from $MSBUILD_DOWNLOAD_URL"
-        curl -sL -o "$MSBUILD_ZIP" "$MSBUILD_DOWNLOAD_URL"
-
+	# Curl is dependency on libcurl4, but some .so's need libcurl3. So using wget
+        wget "$MSBUILD_DOWNLOAD_URL"
+        mv "$MSBUILD_TAR_NAME" "$MSBUILD_ZIP"
```
Comment (radical) [inline cibuild.sh:44]:
```suggestion
	    # Curl is dependency on libcurl4, but some .so's need libcurl3. So using wget
        wget "$MSBUILD_DOWNLOAD_URL" -o "$MSBUILD_ZIP"
```

File: cibuild.sh [hunk 2]
```diff
@@ -141,6 +142,7 @@ THIS_SCRIPT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 PACKAGES_DIR="$THIS_SCRIPT_PATH/packages"
 TOOLS_DIR="$THIS_SCRIPT_PATH/Tools"
 MSBUILD_DOWNLOAD_URL="https://github.com/Microsoft/msbuild/releases/download/mono-hosted-msbuild-v0.03/mono_msbuild_d25dd923839404bd64cc63f420e75acf96fc75c4.zip"
+MSBUILD_TAR_NAME="mono_msbuild_d25dd923839404bd64cc63f420e75acf96fc75c4.zip"
```
Comment (radical) [inline cibuild.sh:145]:
```suggestion
```

**Comment by akoeplinger** [comment]
Closing inactive pull request, please send a new one targetting the `xplat-master` branch if you still need this change :)

**Pull request closed without merging**